services/121-service/package.json

```
@@ -78,7 +78,7 @@
     "ts-node": "^10.9.2",
     "tsconfig-paths": "^4.2.0",
     "twilio": "^5.3.2",
-    "typeorm": "^0.3.20",
+    "typeorm": "npm:@global121/typeorm@1.0.3",
```

Wouldn't it be:
```suggestion
    "@global121/typeorm": "^1.0.3",
```
?

Or is it published to `npm` in a different way?

---

@RubenGeo maybe this is a good invitation to add to the README why we had to do it this way (and to not waste time trying to change it in the future).

---

Good one. Added [this](https://github.com/global-121/121-platform/pull/6120/commits/45f0566e5e9d835929f44f6cca63353505c7669d)